airflow/providers/amazon/aws/example_dags/example_s3_bucket.py

```diff
@@ -49,12 +49,16 @@ def upload_keys():
     tags=['example'],
 ) as dag:
 
-    create_bucket = S3CreateBucketOperator(task_id='s3_bucket_dag_create', region_name='us-east-1')
+    create_bucket = S3CreateBucketOperator(  # type: ignore[call-arg]
+        task_id='s3_bucket_dag_create', region_name='us-east-1'
+    )
 
     # Using a task-decorated function to add keys
     add_keys_to_bucket = upload_keys()
 
-    delete_bucket = S3DeleteBucketOperator(task_id='s3_bucket_dag_delete', force_delete=True)
+    delete_bucket = S3DeleteBucketOperator(  # type: ignore[call-arg]
+        task_id='s3_bucket_dag_delete', force_delete=True
+    )
 
```

Comment:
Chose to ignore the argument checking for each individual operator rather than the entire file (like several other example DAGs) because the check should be performed on the TaskFlow function.  However, the TaskFlow function is unlikely to change so happy to apply ignore check to the entire file as well.